docs/src/guides/moderation.md

```diff
@@ -0,0 +1,50 @@
+Moderation
+==========
+
+`ord` includes a block explorer, which you can run locally with `ord server`.
+
+The block explorer allows viewing inscriptions. Inscriptions are are
+user-generated content, which may be objectionable or unlawful.
+
+It is the responsibility of each individual who runs an ordinal block explorer
+instance to understand their responsibilities with respect to unlawful content,
+and decide what moderation policy is appropriate for their instance.
+
+In order to prevent particular inscriptions from being displayed on an `ord`
+instance, they can be included in a YAML config file, which is loaded with the
+`--config` option.
+
+To hide inscriptions, first create a config file:
+
+```yaml
+hidden:
+- 0000000000000000000000000000000000000000000000000000000000000000i0
```

Comment:
it's probably obvious to most people, but we could be explicit that this is the inscription id